code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm

```diff
@@ -0,0 +1,123 @@
+/datum/caste_datum/facehugger
+	caste_type = XENO_CASTE_FACEHUGGER
+	tier = 0
+	plasma_gain = 0.1
+	plasma_max = 10
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	max_health = XENO_HEALTH_LARVA
+	caste_desc = "Ewwww, that's disgusting!"
+	speed = XENO_SPEED_TIER_8
+
+	evolution_allowed = FALSE
+	can_be_revived = FALSE
+
+/mob/living/carbon/Xenomorph/Facehugger
+	name = XENO_CASTE_FACEHUGGER
+	caste_type = XENO_CASTE_FACEHUGGER
+	speak_emote = list("hisses")
+	icon_state = "Facehugger"
+	icon_size = 32
+	layer = MOB_LAYER
+	see_in_dark = 8
+	tier = 0  //Facehuggers don't count towards Pop limits
+	crit_health = -25
+	gib_chance = 25
+	mob_size = 0
+	counts_for_slots = FALSE
+	refunds_larva_if_banished = FALSE
+	base_actions = list(
+		/datum/action/xeno_action/onclick/xeno_resting,
+		/datum/action/xeno_action/watch_xeno,
+		/datum/action/xeno_action/onclick/xenohide,
+		/datum/action/xeno_action/activable/pounce/facehugger
+	)
+	inherent_verbs = list(
+		/mob/living/carbon/Xenomorph/proc/vent_crawl
+	)
+	mutation_type = "Normal"
+
+	icon_xenonid = 'icons/mob/xenonids/xenonid_crab.dmi'
+
+	var/hugger_life_tick = 0
+
+/mob/living/carbon/Xenomorph/Facehugger/Initialize(mapload, mob/living/carbon/Xenomorph/oldXeno, h_number)
+	icon_xeno = get_icon_from_source(CONFIG_GET(string/alien_hugger))
+	return ..()
+
+/mob/living/carbon/Xenomorph/Facehugger/initialize_pass_flags(var/datum/pass_flags_container/PF)
+	..()
+	if (PF)
+		PF.flags_pass = PASS_MOB_THRU|PASS_FLAGS_CRAWLER
+		PF.flags_can_pass_all = PASS_ALL^PASS_OVER_THROW_ITEM
+
+/mob/living/carbon/Xenomorph/Facehugger/Life(delta_time)
+	hugger_life_tick++
+	if(hugger_life_tick % 2 == 0 && stat != DEAD && !lying)
```

Comment:
so if they're lying anywhere it won't drain?

Reply:
ye

Reply:
Why

Reply:
so they can hide anywhere

Reply:
fairs